### PR TITLE
[ui] Use target information for Schedules

### PR DESCRIPTION
**Pull request by hellendag**
## Summary & Motivation

Use target information for Schedules, same as we do for Sensors. This means adding asset selection secondary queries to the schedules list and detail page.

Note that this isn't shown anywhere on the schedules list, since it would involve adding a new column and I think I need to sort that out with @braunjj.

Screenshot 2024-07-24 at 13 29 53

Screenshot 2024-07-24 at 13 29 45

## How I Tested These Changes

Use sample code location provided by @smackesey that includes anonymous asset jobs on sensors and schedules. Verify that the target appears correctly in both sensor and schedule detail pages, and on the existing sensors list.

### CODE REVIEW

**Comment by smackesey** [comment]
Is this also working in the sensors list? I'm guessing yes but no screenshot so want to confirm.

**Comment by hellendag** [comment]
Demo of the Sensors list, plus the (currently unchanged) behavior of clicking the target link. In a followup I'm going to make a few small changes to how that link behaves.

https://github.com/user-attachments/assets/fea19696-b482-4379-8560-27568a96d6bb

File: js_modules/dagster-ui/packages/ui-core/src/schedules/ScheduleRoot.tsx [hunk 1]
```diff
@@ -55,7 +60,18 @@ export const ScheduleRoot = (props: Props) => {
...
 
-  const refreshState = useQueryRefreshAtInterval(queryResult, FIFTEEN_SECONDS);
+  const selectionQueryResult = useQuery<
+    ScheduleAssetSelectionQuery,
+    ScheduleAssetSelectionQueryVariables
+  >(SCHEDULE_ASSET_SELECTIONS_QUERY, {
+    variables: {scheduleSelector},
+    notifyOnNetworkStatusChange: true,
+  });
+  useBlockTraceOnQueryResult(selectionQueryResult, 'ScheduleAssetSelectionQuery');
```
Comment (salazarm) [inline js_modules/dagster-ui/packages/ui-core/src/schedules/ScheduleRoot.tsx:70]: 👍🏾

File: js_modules/dagster-ui/packages/ui-core/src/workspace/VirtualizedScheduleRow.tsx [hunk 2]
```diff
@@ -81,12 +86,35 @@ export const VirtualizedScheduleRow = (props: ScheduleRowProps) => {
...
+    variables: {
+      scheduleSelector: {
+        repositoryName: repoAddress.name,
+        repositoryLocationName: repoAddress.location,
+        scheduleName: name,
+      },
+    },
+  });
+
+  useBlockTraceOnQueryResult(scheduleAssetSelectionQueryResult, 'ScheduleAssetSelectionQuery');
```
Comment (salazarm) [inline js_modules/dagster-ui/packages/ui-core/src/workspace/VirtualizedScheduleRow.tsx:105]: Would be nice to link to a datadog RUM view of this page before/after to get a sense of the effect on display done.

**Comment by hellendag** [comment]
* **#23233** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23233?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#23208** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23208?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @hellendag and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: js_modules/dagster-ui/packages/ui-core/src/workspace/VirtualizedScheduleRow.tsx (hunk 2 above)
Comment (hellendag) [reply js_modules/dagster-ui/packages/ui-core/src/workspace/VirtualizedScheduleRow.tsx:105]: @salazarm and I confirmed that the queries take roughtly the same amount of time, so it doesn't look like there's any added wait time.

**Review by salazarm — APPROVED** [review]

**Comment by hellendag** [comment]
### Merge activity

* **Jul 26, 1:07 PM CDT**: @hellendag started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/23208).

**Pull request merged**